### DISCUSSION TITLE
Nested Git commands not working in scripts

### DISCUSSION THREAD

**Discussion by aomorishouko** [gitextensions/gitextensions#9323] [Q&A]
When creating a script in Settings > Git Extensions > Scripts that is supposed to execute `git tag -d $(git tag -l)`, this only works if the Command `powershell` is used with the aforementioned Arguments. `git` is not working when given the Arguments `tag -d $(git tag -l)`. Simple Git commands are working with that method, though. Also, when executing the nested command in the integrated console, there is no problem. Can someone explain why that is? Are the scripts using the Windows command shell to execute? I always thought the environment given in Settings > Git > Paths is used to execute commands, am I wrong there? Or is that setting not used in the Scripts context?

Btw: Is it right to assume these scripts are Git Hooks? They don't seem to go to the repo's .git/hooks directory. Where does Git Extensions store them?

**Comment by RussKie** [comment]
> Btw: Is it right to assume these scripts are Git Hooks? They don't seem to go to the repo's .git/hooks directory. Where does Git Extensions store them?

No. The app executes scripts on selected events (see https://github.com/gitextensions/gitextensions/blob/d488e497cc6244483794abedb8e4779495ba861c/GitUI/Script/ScriptEvent.cs). 
The scripts are stored in your main gitextensions.config (C:\Users\<user name>\AppData\Roaming\GitExtensions\GitExtensions\GitExtensions.settings)
